src/Filters/coefficients.jl

```diff
@@ -380,3 +380,19 @@ SecondOrderSections(f::FilterCoefficients) = SecondOrderSections(ZeroPoleGain(f)
 *(f1::Biquad, fs::Biquad...) = SecondOrderSections([f1, fs...], 1)
 *(f1::SecondOrderSections, f2::Biquad) = SecondOrderSections([f1.biquads; f2], f1.g)
 *(f1::Biquad, f2::SecondOrderSections) = SecondOrderSections([f1; f2.biquads], f2.g)
+function inv(f::FilterCoefficients)
```

Comment:
I added a "inv" function because by default taking f::FilterCoefficients ^ -1 invoked Base.inv and caused an error.